app/src/main/java/com/example/riden/ParseApplication.java

```diff
@@ -2,17 +2,22 @@
 
 import android.app.Application;
 
+import com.example.riden.models.Ride;
+import com.example.riden.models.User;
 import com.parse.Parse;
 import com.parse.ParseObject;
 
 public class ParseApplication extends Application {
+
     @Override
     public void onCreate() {
         super.onCreate();
+        ParseObject.registerSubclass(User.class);
+        ParseObject.registerSubclass(Ride.class);
```

Comment:
Was this sufficient to query rides ? What was the issue that caused 0 rides to be returned when we were debugging ?